pkg/manifest/build.go

```diff
@@ -36,12 +34,30 @@ type BuildrootFromPackages struct {
 	packageSpecs []rpmmd.PackageSpec
 
 	containerBuildable bool
+
+	// XXX: disableSelinux controlls if selinux should be disabled
+	// for the given buildroot. This is currently needed for
+	// bootstraped buildroot containers because most of the
+	// boostrap containers do not include setfiles(8). A
+	// reasonable fix is to teach osbuild to chroot into the
+	// buildroot itself when running setfiles. Once osbuild has
+	// this then this option would become "useChrootSetfiles"
```

Comment:
I'll assume we have requested `setfiles` to be included, if so, do we have tracking issues for this?

Reply:
I created  https://github.com/osbuild/osbuild/issues/2052 for this - I am not sure how likely it is that we can convince upstreams but allowing to chroot in setfiles seems okay to me (and not too much work)